sandbox_rpc.go

```diff
@@ -28,14 +41,22 @@ type Response struct {
 
 // A single unit of operation
 type StoredDataQuantum struct {
-	Bucket string        `json:"bucket"`
-	Data   []*StoredData `json:"data"`
+	Bucket    string        `json:"bucket"`
+	Data      []*StoredData `json:"data"`
+	Batch     *Batch        `json:"batch,omitempty"`
+	QueryTime float64       `json:"query_time,omitempty"`
```

Comment:
Can we not encapsulate these as well into something like `QuantumMeta`?

Reply:
Now encasulated.